scripts/maps_gcp.py

```diff
@@ -0,0 +1,52 @@
+"""GCP script for MAPS from complete gnomAD set"""
+
+import hail as hl
+import gnomad.utils.vep
+
+from prepare_gnomAD import get_worst_consequence_with_non_coding, prepare_ht
+from maps_score import maps
+
+if __name__ == '__main__':
+    import argparse
+
+    parser = argparse.ArgumentParser(description='Compute MAPS score')
```

Comment:
Suggesting to add `help` parameters to the `add_argument` functions. Would improve the readability, otherwise hard to track down what does e.g. `context_ht` contain.

Reply:
Added some help statements, however I'm not completely sure how exactly the `context_ht` is defined and what it contains. I'm trying to contact the authors of the hail implementation of the MAPS score, so I'll hopefully know soon